lib/rover/data_frame.rb

```diff
@@ -416,7 +416,7 @@ def plot(x = nil, y = nil, type: nil)
           raise "Cannot determine type. Use the type option."
         end
       end
-      data = self[[x, y]]
+      data = self[group.nil? ? [x, y] : [x, y, group]]
 
       case type
       when "line", "area"
```

Comment:
Let's add it to all chart types. For charts that don't support it (I think just pie right now), we can raise an `ArgumentError`.

Reply:
But if we use `self[[x, y, group]]`, and don't pass the `group` argument, it throws an error.

Reply:
Yeah, that line looks good (comment was meant for `case` statement).